### PR TITLE
#21 Set non-zero exit status when some files failed to be encrypted/decrypted or some groups admins/members failed to be removed/added

### PR DESCRIPTION
**Pull request by ernieturner**

### CODE REVIEW

File: test/commands/group/removeadmins.test.ts [hunk 1]
```diff
@@ -1,66 +0,0 @@
-import {expect} from "@oclif/test";
```
Comment (clintfred) [inline test/commands/group/removeadmins.test.ts]: Was removal of this test file intentional?

File: src/commands/group/addadmin.ts [hunk 2]
```diff
@@ -52,6 +52,9 @@ export default class AddAdmin extends Command {
             .group.addAdmins(groupID, flags.users as string[])
             .then((addResult) => {
                 this.log(`\n${this.buildResultTable(addResult).toString()}\n`);
+                if (addResult.failed.length > 0) {
+                    this.error(`Failed to add ${addResult.failed.length} admin(s).`);
```
Comment (clintfred) [inline src/commands/group/addadmin.ts]: Should we tell this which admin failed?

Comment (ernieturner) [reply src/commands/group/addadmin.ts]: We already do. On line 54 we're displaying a table that shows the list of success and failures. This added log was just to cause that error to be obvious and exit with a non-zero code.

File: test/commands/group/removeadmins.test.ts (hunk 1 above)
Comment (ernieturner) [reply test/commands/group/removeadmins.test.ts]: Ya, I'm not sure what happened, but we already have a `removeadmin.test.ts` (edited just above this) so I'm not sure why we had a duplicate test file for the same functionality.

**Comment by ernieturner** [comment]
@cjyar Unfortunately we don't have any tests for the encrypt/decrypt methods as of yet. We have a ticket to add that work, but mocking out all the filesystem level stuff was enough of a blocker that we delayed that work and haven't yet prioritized it.

**Pull request merged**